percona-server.57/Dockerfile-dockerhub

```diff
@@ -0,0 +1,56 @@
+FROM centos:7
+MAINTAINER Percona Development <info@percona.com>
+
+RUN groupdel input && groupadd -g 999 mysql
+RUN useradd -u 999 -r -g 999 -s /sbin/nologin \
+		-c "Default Application User" mysql
+
+# check repository package signature in secure way
+RUN export GNUPGHOME="$(mktemp -d)" \
+	&& gpg --keyserver ha.pool.sks-keyservers.net --recv-keys 430BDF5C56E7C94E848EE60C1C4CBDCDCD2EFD2A \
+	&& gpg --export --armor 430BDF5C56E7C94E848EE60C1C4CBDCDCD2EFD2A > ${GNUPGHOME}/RPM-GPG-KEY-Percona \
+	&& rpmkeys --import ${GNUPGHOME}/RPM-GPG-KEY-Percona /etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-7 \
+	&& curl -L -o /tmp/percona-release.rpm http://www.percona.com/downloads/percona-release/redhat/0.1-6/percona-release-0.1-6.noarch.rpm \
+	&& rpmkeys --checksig /tmp/percona-release.rpm \
+	&& yum install -y /tmp/percona-release.rpm \
+	&& rm -rf "$GNUPGHOME" /tmp/percona-release.rpm
+
+# install exact version of PS for repeatability
+ENV PERCONA_VERSION 5.7.23-23.1.el7
+
+RUN yum install -y \
+		Percona-Server-server-57-${PERCONA_VERSION} \
+		Percona-Server-tokudb-57-${PERCONA_VERSION} \
+		Percona-Server-rocksdb-57-${PERCONA_VERSION} \
+		jemalloc \
+		which \
+		policycoreutils \
+	&& yum clean all \
+	&& rm -rf /var/cache/yum /var/lib/mysql
+
+# purge and re-create /var/lib/mysql with appropriate ownership
+RUN /usr/bin/install -m 0775 -o 999 -g 0 -d /var/lib/mysql /var/run/mysqld /docker-entrypoint-initdb.d \
```

Comment:
minor nitpick: you can user `-o mysql -g root` as the user/group already created in this very Dockerfile.

Reply:
fixed